jmx/plugins/org.jboss.tools.jmx.jolokia/pom.xml

```diff
@@ -19,7 +19,7 @@
 	<properties>
 		<jolokia.client.version>1.4.0</jolokia.client.version>
 		<json.simple.version>1.1.1</json.simple.version>
-		<httpcomponents.version>4.5.2</httpcomponents.version>
+		<httpcomponents.version>4.5.13</httpcomponents.version>
```

Comment:
I'm lost why do you keep this and copy dependencies if taken from TP ?

Reply:
removed.